DevCycle/Models/UserConfig.swift

```diff
@@ -147,7 +147,7 @@ public struct Feature {
     }
 }
 
-public struct Variable {
+public class Variable {
```

Comment:
any reason to change this from a struct to class?

Reply:
yeah, I should've mentioned this. The NSMapTable doesn't work with `Any` type, it works with `AnyObject` which doesn't include Structs

Reply:
When do you ever need to store this `struct` locally, I thought we only need to store the `DVCVariable` class?

Reply:
I think I was doing it incorrectly originally. I changed it back.